runelite-client/src/main/java/net/runelite/client/plugins/itemprices/ItemPricesOverlay.java

```diff
@@ -246,9 +251,36 @@ private String stackValueText(int qty, int gePrice, int haValue)
 			}
 		}
 
+		if (haProfit > 0)
+		{
+			Color haColor = haProfitColor(haProfit);
+
+			itemStringBuilder.append("</br>");
+			itemStringBuilder.append("HA P/L: ")
```

Comment:
I would probably make this say like HA Profit always (HA P/L says nothing and I dont think labeling profit & loss like that is even a thing anywhere)